From a1f18876d5f7122c9f94af9a84f1324f50dba0ed Mon Sep 17 00:00:00 2001
From: Jan Janssen <medhefgo@web.de>
Date: Tue, 1 Nov 2022 09:43:32 +0100
Subject: [PATCH] coverage: Mark _coverage__exit as noreturn

../src/basic/coverage.h:15:48: warning: function '_coverage__exit' could
be declared with attribute 'noreturn' [-Wmissing-noreturn]

(cherry picked from commit 0bab5534b334677652bb69fe15eaa54ce84cbe7d)

Related #2138081
---
 src/basic/coverage.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/coverage.h b/src/basic/coverage.h
index 3ef02cf70e..640bddc485 100644
--- a/src/basic/coverage.h
+++ b/src/basic/coverage.h
@@ -12,7 +12,7 @@
 extern void _exit(int);
 extern void __gcov_dump(void);
 
-static inline void _coverage__exit(int status) {
+static inline _Noreturn void _coverage__exit(int status) {
         __gcov_dump();
         _exit(status);
 }
